client/src/context/authPanel/provider.tsx

```diff
@@ -0,0 +1,36 @@
+import { /*type*/ AuthPanelOption } from 'components/AuthPanel/types';
+
+import React, { useReducer } from 'react';
+import { AuthPanelContext, initialState } from './state';
+import Reducer from './reducer';
+import ActionTypes from './actions';
+
+interface Props {
+  children: React.ReactNode | React.ReactNode[];
+}
+
+const Provider = ({ children }: Props) => {
```

Comment:
If u type `Provider` as `React.FC` u don't have to worry about defining children in the props

Reply:
Whaaa that's sick

Reply:
That's sick! Will create a refactor PR